### PR TITLE
Update README.md formatted Customize Buttons List

### PR DESCRIPTION
**Pull request by alextsg**
Formatted Customize Buttons List for consistency(periods at end of items) and wording.

BEFORE:
Clone the Buttons repository.
Make sure you have node.js installed.
From the command line cd into the root for the Buttons directory
Run npm install or sudo npm install (depending on your system permissions).
On the command line run grunt dev, this will open a browser with Buttons
Locate scss in the root directory
You can modify the _options.scss where you can customize colors, typography, and …
Anytime you save your changes the Buttons showcase page will live reload with your changes!

AFTER:
Clone the Buttons repository.
Make sure you have node.js installed.
From the command line cd into the root for the Buttons directory.
Run npm install or sudo npm install (depending on your system permissions).
On the command line run grunt dev; this will open a browser with Buttons.
Locate scss in the root directory.
You can modify the _options.scss file where you can customize colors, typography, etc.
Anytime you save your changes the Buttons showcase page will live reload with your changes!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,11 +49,11 @@ We've made some major improvements to the Buttons library. In order to integrate
 ## Customize Buttons (Recommended uses Sass & Autoprefixer)
 0. Clone the Buttons repository.
 0. Make sure you have [node.js](http://nodejs.org/) installed.
-0. From the command line cd into the root for the Buttons directory
+0. From the command line ```cd``` into the root for the Buttons directory.
```
Comment (alexwolfe) [inline README.md]: Inline code snippets generally have one tick mark <pre> `cd`</pre>

**Comment by alexwolfe** [comment]
@alextsg I just had one comment but otherwise looks good!

**Comment by alextsg** [comment]
Changed!

**Comment by roblevintennis** [comment]
@alextsg thanks!

**Pull request merged**